### ISSUE TITLE
Add support for custom helm command line options

### ISSUE THREAD

**Issue by tjcrone** [berkeley-dsep-infra/hubploy#34]
Sometimes I need to include --force because of a failed deployment on the first deployment, or include the --timeout option to helm. When I need to do that I [hardcode these](https://github.com/tjcrone/hubploy/commit/ef622c0ba38619f11863727779ef3bb36fb8e639), and use my version of hubploy. I'm wondering if we could include an option for custom helm options instead? @yuvipanda, I'm happy to make these changes and PR if you suggest a preferred way of doing this. Thanks!

**Comment by yuvipanda** [comment]
@tjcrone yeah, I think a '--helm-options' flag maybe? That could just get passed in to helm.

**Comment by tjcrone** [comment]
Okay. I'm going to have to think about how to pass a list of flags and values for another program through the hubploy argparse with a --helm-options flag. Maybe a list of strings. Maybe a dict. I have not passed a set of options for another program to argparse before. If you have a preferred method, please let me know.

**Comment by tjcrone** [comment]
Kindof need a way to get **kwargs working with argparse and the CLI. Googling ...

**Comment by yuvipanda** [comment]
I'd suggest taking it in as a single string, splitting it with [shlex.split](https://docs.python.org/3/library/shlex.html#shlex.split), and then passing it to helm.

**Comment by tjcrone** [comment]
Another option would be to add "force=False" and "timeout=300" as separate argparse options to hubploy, and then pass them on to helm. I kindof like this idea better. What do you think? If you do not like this, I will do it as a string like you suggest.
